modules/nf-core/bcftools/pluginimputeinfo/meta.yml

```
@@ -0,0 +1,81 @@
+---
+name: "bcftools_pluginimputeinfo"
+description: Adds imputation information metrics to the INFO field based on selected FORMAT tags. Only the IMPUTE2 INFO metric from FORMAT/GP tags is currently available.
+keywords:
+  - impute-info
+  - bcftools
+  - imputation
+  - metrics
+  - tags
+  - VCF
```

```suggestion
  - vcf
```

https://nf-co.re/docs/guidelines/components/modules#keyword-formatting